vms/components/multisig/camino_multisig.go

```diff
@@ -38,6 +45,14 @@ func (ma *Alias) VerifyState() error {
 	return ma.Verify()
 }
 
+func (ma *Alias) Less(other *Alias) bool {
+	return ma.ID.Less(other.ID)
+}
+
+func (mn *AliasWithNonce) Less(other *AliasWithNonce) bool {
+	return mn.Alias.Less(&other.Alias) && mn.Nonce < other.Nonce
```

Comment:
this will lead to case, where 2 different aliases will be sorted wrongly
for example: alias1.ID < alias2.ID, then we expect alias1 < alias2
but alias1.Nonce == 0 == alias2.Nonce, so alias1.Nonce not < alias2.Nonce, so Less method will return false

Reply:
Nice catch 👍